web/yo/app/scripts/controllers/gene.js

```diff
@@ -599,7 +612,7 @@ angular.module('oncokbApp')
                         evidencesAllUsers[userName].deletedEvidenceModels.push(['mutation', mutation]);
                         continue;
                     }
-                    if (mutation.oncogenic_review.get('mostRecentUpdatedBy') === userName) {
+                    if (mutation.oncogenic_review.get('mostRecent') && mutation.oncogenic_review.get('mostRecent').by.getText() === userName) {
```

Comment:
There is no get() listed in the Collaborative method document. I just tried to print out this.gene.name.get(), and it gave me error saying "Uncaught TypeError: this.gene.name.get is not a function"